### DISCUSSION TITLE
Synchronize the theme with system

### DISCUSSION THREAD

**Discussion by aidistan** [FreshRSS/FreshRSS#4508] [Show and tell]
Hi,

I read a lot on my phone at night and got tired of switching the theme between light and dark. That's why I made [this extension](https://github.com/aidistan/freshrss-extensions): 

```json
{
    "name": "Sync with System Theme",
    "description": "Synchronize FreshRSS's theme with your system theme",
    "url": "https://github.com/aidistan/freshrss-extensions",
    "type": "user"
}
```

It works fine on both desktop and mobile. 

Here is a short demo video:

https://user-images.githubusercontent.com/3037661/185191808-af1e375c-e9e5-41ca-942a-8b714f50a774.mp4

Hope it will help somebody else. Feel free to leave any comment and suggestion.

**Comment by Frenzie** [comment]
Sounds cool. :-)

A quick and simple not as nice semi-equivalent could be something like this (or with `light` for one of the dark themes).
```css
@media (prefers-color-scheme: dark) {
  html, img { filter: invert(1) }
}
```
